gulpfile.js

```diff
@@ -65,6 +65,22 @@ function test() {
 gulp.task('build-test', ['build'], test);
 gulp.task('test', test);
 
+gulp.task('checkImports', function (cb) {
+    var checkProcess = child_process.fork(path.join(__dirname, "tools", "checkCasing.js"),
+        {
+            cwd: path.resolve(__dirname, "src"),
+            stdio: "inherit"
+        });
+    checkProcess.on("error", cb);
```

Comment:
Yes; if an error event happens then something has gone wrong and I'm fine with it failing the task until we fix it, and if the process end is called afterwards then it'll just fail it even harder.